media/server/main/interface/IControlServerInternal.h

```diff
@@ -61,11 +61,11 @@ class IControlServerInternalFactory : public IControlFactory
     createControlServerInternal(int id, const std::shared_ptr<IControlClientServerInternal> &client) const = 0;
 };
 
-class IControlServerInternal : public IControl
```

Comment:
All ServerInternal objects inherit from public ones. Please keep it. What if we add a new method to IControl in future?

Reply:
Then perhaps it should inherit from something else? Perhaps a public interface class that has no methods?
I think that it's wrong that this inherits from IControl when nothing actually needs ServerInternal to implement any of the methods that are required by the IControl interface?

But... if you have plans for something in the future then I'll leave it..

Reply:
That's why we keep that inheritance - to be ready for public interface changes, which have to be handled both on server and client side ;)